src/riscv/lib/src/jit/builder/control_flow_graph.rs

```diff
@@ -186,6 +211,19 @@ where
         }
     }
 
+    /// Get outgoing outcomes from a node, sorted by probability, with most probable first.
+    fn sorted_outgoings(&self, source_node: InstrId) -> IntoIter<&OutcomeId> {
```

Comment:
Nit: You don't have to return an exact iterator type.

```suggestion
    fn sorted_outgoings(&self, source_node: InstrId) -> impl Interator<Item = &OutcomeId> {
```

Reply:
done [here](https://github.com/tezos/riscv-pvm/compare/1cb98413e0bb77c9705a29efa37545a2df2ad730..9d0afbd4dec2013d6f6dfce66a62c2e511032a35)